docs/getting-started/dockstore-tools.rst

```diff
@@ -13,46 +13,39 @@ Tutorial Goals
 
 Assumptions
 -----------
-This document assumes you have already have a tool ready to register. You do not need to have a Dockerfile associated with it unless you are using one our legacy registration methods (more on that below).
+This document assumes you already have a tool ready to register. 
 
-If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. Of course, you can always follow along with your own unique tools too, which may or may not be associated with an autobuilding Docker image.
+If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. You can also follow this tutorial if your tool is not associated with an autobuilding Docker image.
```

Comment:
Getting started with CWL has since changed; the old tutorial which included a Dockerfile and autobuilding is now gone. (Well, strictly speaking, [moved](https://docs.dockstore.org/en/stable/advanced-topics/legacy/legacy-getting-started-with-cwl.html).) I propose changing this paragraph to the following.


> If you followed :doc:\`Getting Started With CWL </getting-started/getting-started-with-cwl>\` to create a CWL tool, you will now have a \`\`Dockstore.cwl\`\` in GitHub and have linked your accounts to Dockstore. This will allow you to use the Dockstore GitHub App to register your tool. Of course, you can always follow along with your own unique tools too. Regardless of how you made your tool, this tutorial will assume you are using CWL. For all other languages, :doc:\`please see our documentation on workflows instead </getting-started/dockstore-workflows>\`.

Reply:
On a second reading, I think the third and forth sentence that I wrote just above could be improved... feel free to play with that paragraph until it sounds right, I just think we need to acknowledge that the current Getting Started With CWL does not involve Dockerfiles.